types/openlayers/index.d.ts

```diff
@@ -9,6 +9,7 @@
 //                 Yair Tawil <https://github.com/yairtawil>
 //                 Pierre Marchand <https://github.com/pierremarc>
 //                 Hauke Stieler <https://github.com/hauke96>
+//                 Guillaume Beraudo <https://github.com/gberaudo>
```

Comment:
According to the DefinitelyTyped, adding yourself as an author will cause you to be notified whenever someone makes a pull request or issue about the package. Is it really what you want?

Reply:
yes